DigiMeSDK/Core/Classes/Network/DMERequestFactory.m

```diff
@@ -80,6 +83,88 @@ - (NSURLRequest *)fileRequestWithId:(NSString *)fileId sessionKey:(NSString *)se
     return request;
 }
 
+- (NSURLRequest *)dataPushWithPostboxId:(NSString *)postboxId dataToPush:(NSData *)data headerParameters:(NSDictionary *)headers
+{
+    NSURL *url = [NSURL URLWithString:[NSString stringWithFormat:@"%@/postbox/%@", self.baseUrlPath, postboxId]];
+    NSMutableURLRequest *request = [NSMutableURLRequest requestWithURL:url cachePolicy:NSURLRequestUseProtocolCachePolicy timeoutInterval:self.config.globalTimeout];
+    
+    [request setCachePolicy:NSURLRequestReloadIgnoringLocalCacheData];
+    [request setHTTPMethod:@"POST"];
+    
+    NSString *boundary = [self generateBoundaryString];
+    NSString *metadata = [[headers[@"metadata"] stringByReplacingOccurrencesOfString:@"\n" withString:@""]stringByReplacingOccurrencesOfString:@"\r" withString:@""];
+    NSString *symmetricalKey = [[headers[@"symmetricalKey"] stringByReplacingOccurrencesOfString:@"\n" withString:@""]stringByReplacingOccurrencesOfString:@"\r" withString:@""];
+    
+    NSString *contentType = [NSString stringWithFormat:@"multipart/form-data; boundary=%@", boundary];
+    [request setValue:contentType forHTTPHeaderField: @"Content-Type"];
+    [request setValue:@"application/json" forHTTPHeaderField:@"Accept"];
+    [request setValue:headers[@"sessionKey"] forHTTPHeaderField: @"sessionKey"];
+    [request setValue:symmetricalKey forHTTPHeaderField: @"symmetricalKey"];
+    [request setValue:headers[@"iv"] forHTTPHeaderField: @"iv"];
+    [request setValue:metadata forHTTPHeaderField: @"metadata"];
+    
+    NSMutableData *body = [NSMutableData data];
+    [body appendData:[[NSString stringWithFormat:@"--%@\r\n", boundary] dataUsingEncoding:NSUTF8StringEncoding]];
```

Comment:
`\r\n are used to terminate lines in Windows and DOS text files.`.

I believe just `\n` should be enough.